3d Space Shooter/Assets/TextMesh Pro/Examples & Extras/Fonts/Oswald Bold - OFL.txt

```diff
@@ -0,0 +1,92 @@
+Copyright (c) 2011-2012, Vernon Adams (vern@newtypography.co.uk), with Reserved Font Names 'Oswald'
```

Comment:
Whats with all the different licenses? How many fonts are we using? Ideally we should only use two to three in the entire game, so we don't need all these licenses.

Reply:
These could have been added automatically.